File: Techblazers-Part2/LettersAndNumbers.java
import java.util.Scanner;

public class LettersAndNumbers {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        System.out.print("Enter a sentence: ");
        String userInput = input.next();
        
        int countIntegers = 0;
        int countLetters = 0;
        
        for (char character : userInput.toCharArray()) {
            if (Character.isDigit(character)) {
                countIntegers++;
            } else if (Character.isLetter(character)) {
                countLetters++;
            }
        }
        
        System.out.println("Letters: " + countLetters);
        System.out.println("Numbers: " + countIntegers);
    }
}

File: Techblazers-Part2/lettersAndNumbers.py
users_input = input("Enter a sentence: ")

count_integers = 0
count_letters = 0

for character in users_input:
	if character.isdigit():
		count_integers += 1
	elif character.isalpha():
		count_letters += 1
print("Letters:", count_letters)
print("Numbers:", count_integers)